src/main/scripts/src/app/collection/collection.module.ts

```diff
@@ -27,7 +27,6 @@ import {SimpleNotificationsModule} from 'angular2-notifications/dist';
 @NgModule({
   imports: [
     SharedModule,
-    SimpleNotificationsModule,
```

Comment:
When this is removed, the application cannot be loaded after AOT compilation.

Reply:
It does the opposite for me.
With the module imported in `collection.module.ts` it fails to build (both regular and AOT) and without it both run with no errors.

Reply:
Try to remove `node_modules` and run `npm install` again. Does it work now?

Reply:
Same.
Although changing `from 'angular2-notifications/dist';` to `from 'angular2-notifications';` fixed the issue as well for me.

Reply:
The changes you are suggesting do not work with AOT compilation. The current solution works for me using both JIT and AOT compilation.

Reply:
If it worked for @SquattingPenguin without removing `SimpleNotificationsModule` import, it should not be removed and we can discuss your problem somewhere else since it is not related to this PR at all. (Those changes were introduced before.)